static/.htaccess

```diff
@@ -8,6 +8,9 @@ Options -Indexes -MultiViews
 
 # Redirect pages - https://httpd.apache.org/docs/current/mod/mod_rewrite.html#rewriterule
 
+RewriteRule ^docs/next/apis-clients/operate-api(.*)$ /docs/next/apis-clients/operate-api/operate-api-overview [R=301,L]
+RewriteRule ^docs/apis-clients/operate-api(.*)$ /docs/apis-clients/operate-api/operate-api-overview [R=301,L]
```

Comment:
```suggestion
RewriteRule ^docs/next/apis-clients/operate-api/$ /docs/next/apis-clients/operate-api/overview/ [R=301,L]
```

@christinaausley A few things I noticed about these rules: 

1. The first one redirects to the incorrect URL. `operate-api-overview` is the _id_ of the target page, but redirect rules should use the _url_ of the target page. In this case, the target is defined in that page's `slug` property as `/apis-clients/operate-api/overview`.
2. The first rule is capturing too much, with the `(.*)` at the end of its "match" argument. That actually causes a redirect loop --- the first request, it redirects from .../operate-api to .../operate-api/overview, and that request redirects from .../operate-api/overview to .../operate-api, then back to overview, then back to no overview, etc. This is because the `(.*)` matches _everything_ after the `operate-api` text, which includes child paths, like the `overview` page is. 
   
   In our case, we really only want to redirect the apis-clients/operate-api/ URL specifically -- so I'd remove the `(.*)` from the end.
4. In general, the right side of these rules should probably end in a slash. It's not required, but it saves one more redirect. The way this site is set up, anything that ends without a slash gets redirected to that same url _followed by a slash_, to serve the `index.html` in that directory. (That behavior isn't declared in this .htaccess; it's built into the webserver itself.) 

   If we redirect to a URL that includes the slash, we save them that one hop.
7. The second rule is, I think, the rule that most build errors were from. The redirect rule is being applied to the "current" version, but these changes are only in the "next" version. So it's redirecting to a page that doesn't yet exist for the "current" version.